src/comparisons/elements/remove/jquery.js

```diff
@@ -1 +1,3 @@
 $(el).remove();
+
+$('el').remove();
```

Comment:
Tiniest of nits: we use the convention of "selector" everywhere where a selector might be appropriate, so to remain consistent we could probably change this to:
```suggestion
$(selector).remove();
```